Files/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -411,5 +437,36 @@ private async void FileList_ItemClick(object sender, ItemClickEventArgs e)
                 ParentShellPageInstance.InteractionOperations.OpenItem_Click(null, null);
             }
         }
+
+        #region Animation
+
+        private CompositionAnimationGroup createOffsetAnimation()
```

Comment:
```suggestion
        private CompositionAnimationGroup CreateOffsetAnimation()
```

Reply:
This should be resolved wtih a4237e4c5e8e80d29e735bd911951973d54cf3e2.